pkgs/development/python-modules/gasp/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildPythonPackage, fetchPypi, tkinter, pygame }:
+
+buildPythonPackage rec {
+  pname = "gasp";
+  version = "0.6.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-3h90H+E0pvDtpT/eQUsLjnnFholbxxk7K66exB6W3vo=";
+  };
+
+  format = "pyproject";
+
+  doCheck = false;
```

Comment:
The upstream repo contains [tests](https://codeberg.org/GASP/GASP_Code/src/branch/main/tests). Could you check if the tests can be executed by using the source from Codeberg instead of PyPI?

Please add `pythonImportsCheck` as additional safe-guard.

Reply:
The tests don't run because they require a display to exist. Added a `pythonImportsCheck`

Reply:
Can we fake it with xvfb-run?

Reply:
@ModdedGamers whats with this?

Reply:
I'm looking into it.

Reply:
The tests directory is not included with `fetchPyPi`, therefore the tests cannot be run. Any solutions(using `fetchFromGitea` `fetchgit`, etc)?

Reply:
Sure. We do that very often if the pypi tarball does not contain everything we need.